etc-conf/rhsm.conf

```diff
@@ -100,6 +100,8 @@ disable = 0
 auto_registration = 0
 # Interval to run auto-registration (in minutes):
 auto_registration_interval = 60
+# Disable auto attach. Only works when used with auto_registration.
+disable_auto_registration = 1
```

Comment:
`disable_auto_registration`? I think that intention was to write `disable_auto_attach` :-)